source/jobs/creating-jobs.txt

```diff
@@ -75,7 +75,9 @@ Steps
           - Leaving database name blank loads schemas from the default database.
         * - Sybase ASE
           - Leaving database name blank loads schemas from the default database.
-  
+        * - DB2
+          - Leaving database name blank loads schemas from the default database. (need confirmation)
```

Comment:
Need confirmation on this also @alwaysbemark  @tomhollander

Reply:
They have to specify a database, not pointing to one will lead to a syntax error on the UI